flake.lock

```diff
@@ -78,11 +81,11 @@
     },
     "nixpkgs": {
       "locked": {
-        "lastModified": 1729805696,
-        "narHash": "sha256-FArm/EIAbykrhtWxWKT1QXIg+dD44joehXZWdY12WKc=",
+        "lastModified": 1735651292,
+        "narHash": "sha256-YLbzcBtYo1/FEzFsB3AnM16qFc6fWPMIoOuSoDwvg9g=",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "50286248f2d7283682bdd47ba14af33a9233b88b",
+        "rev": "0da3c44a9460a26d2025ec3ed2ec60a895eb1114",
```

Comment:
Do we still need the override in build overlay after bump nixpkgs?

Reply:
actually bump nixpkgs is optional after we have newer overlay :D

Reply:
then let's wait for the overrides get included?